src/read.c

```diff
@@ -1181,7 +1184,13 @@ void ReadFuncExpr (
 		SyntaxWarning("arg used not as the last argument");
 	      }
 
+        if (isvarg) {
+            SyntaxError("Only final argument can be variadic");
+        }
+    
```

Comment:
The indention here is different from the surrounding code (which seems to mix tabs and spaces). I think it is best to match the indenting mode of the surrounding code, to keep the code readable. For now, that is (we can fix the indention globally later on, perhaps during GAP days).